MotorControl/MotorControl/Resources/Finger_Tapping.json

```diff
@@ -43,7 +43,7 @@
                 "identifier":"fingerTappingInstructions",
                 "type":"instruction",
                 "title":"You will finger tap for 30 seconds with each hand",
-                "text":"Speed of finger tapping can reflect severity of motor symptoms in Parkinson’s disease. This activity measures your tapping speed for each hand. Your medical provider may measure this differently.",
+                "text":"This activity measures your tapping speed for each hand. Your medical provider may measure this differently.",
```

Comment:
same here as above, adding that we are also measuring accuracy b/c that extracted feature is important for tremor as well